[WIP] Add Feast integration docs (#10)
scylladb/scylladb-feature-store: docs/source/feast-scylladb-online-store.md

# Integrate ScyllaDB and Feast

> Feast is a popular open-source feature store for production ML. You can use several online stores when using Feast, including ScyllaDB. ScyllaDB, a low-latency and high-performance database, serves perfectly as an online store. In this section, you'll see how you can integrate your ScyllaDB Cloud database into Feast as an online store.

Reviewer: ```suggestion
[Feast](https://feast.dev/) is a popular open-source feature store for production ML. You can use several online stores when using Feast, including ScyllaDB. ScyllaDB, a low-latency and high-performance database, serves perfectly as an online store. In this section, you'll see how you can integrate your ScyllaDB Cloud database into Feast as an online store.
```

> Feast is a popular open-source feature store for production ML. You can use several different online stores when you use Feast, including ScyllaDB. ScyllaDB, being a low-latency and high-performance database, serves perfect as an online store. In this section you'll see how you can get started with integrating your ScyllaDB Cloud database into Feast as an online store. 

Reviewer: ```suggestion
Feast is a popular open-source feature store for production ML. You can use several online stores when using Feast, including ScyllaDB. ScyllaDB, a low-latency and high-performance database, serves perfectly as an online store. In this section, you'll see how you can integrate your ScyllaDB Cloud database into Feast as an online store.
```

> If you want to learn more specifically about Feast, head over to the [Feast documentation](https://docs.feast.dev/).

Reviewer: ```suggestion
If you want to learn more about Feast, head to the [Feast documentation](https://docs.feast.dev/).
```
